### PR TITLE
4.0.0-beta.2

### PR DESCRIPTION
**Pull request by bastianallgeier**

### CODE REVIEW

File: panel/src/components/Forms/Layouts/Layout.vue [hunk 1]
```diff
@@ -161,7 +161,12 @@ export default {
 .k-layout:focus {
 	outline: 0;
 }
-
+/* override structure field style for sections */
+/* TODO: Remove in 4.0 */
```
Comment (lukasbestle) [inline panel/src/components/Forms/Layouts/Layout.vue]: This got merged in by accident.

**Comment by bastianallgeier** [comment]
@afbora that's great! Thanks for thinking about that part.

**Review by lukasbestle — COMMENTED** [review]
Looks like the “update dependencies after merge” commit actually reverted back to lower versions.

**Comment by bastianallgeier** [comment]
@lukasbestle I just fixed it.

**Pull request merged**